packages/contracts-bedrock/scripts/go-ffi/README.md

```diff
@@ -0,0 +1,50 @@
+# `ctb-go-ffi`
+
+A lightweight binary for utilities accessed via `forge`'s `ffi` cheatcode in the `contracts-bedrock` test suite.
+
+<pre>
+├── go-ffi
+│   ├── <a href="./bin.go">bin</a>: `go-ffi`'s binary
+│   ├── <a href="./trie.go">trie</a>: Utility for generating random merkle trie roots / inclusion proofs
+│   └── <a href="./differential-testing.go">diff-testing</a>: Utility for differential testing Solidity implementations against their respective Go implementations.
+</pre>
+
+## Usage
+
+To build, run `pnpm build:go-ffi` from this directory or the `contract-bedrock` package.
+
+### Available Modes
+
+#### `diff`
+
+> **Note**
+> Variant required for diff mode.
```

Comment:
what does Variant mean in this context?

Reply:
OK, I see it's how the command is invoked. Perhaps give an example of how to call the binary here. 

Could also show how to use it in a forge test, or refer to an example in the repo.